views/js/qtiCreator/widgets/interactions/Widget.js

```diff
@@ -86,8 +87,27 @@ define([
      */
     InteractionWidget.createToolbar = function(options){
 
+        const classTitles = {
+            choiceInteraction: __('Choice Interaction'),
+            orderInteraction: __('Order Interaction'),
+            associateInteraction: __('Associate Interaction'),
+            matchInteraction: __('Match Interaction'),
+            hottextInteraction: __('Hottext Interaction'),
+            gapMatchInteraction: __('Gap Match Interaction'),
+            sliderInteraction: __('Slider Interaction'),
+            extendedTextInteraction: __('Extended Text Interaction'),
+            uploadInteraction: __('File Upload Interaction'),
+            mediaInteraction: __('Media Interaction'),
+            hotspotInteraction: __('Hotspot Interaction'),
+            graphicOrderInteraction: __('Graphic Order Interaction'),
+            graphicAssociateInteraction: __('Graphic Associate Interaction'),
+            graphicGapMatchInteraction: __('Graphic Gap Match Interaction'),
+            selectPointInteraction: __('Select Point Interaction')
+        }
+        const getQtiClassTitle = qtiClass => classTitles[qtiClass] || qtiClass;
```

Comment:
What do you think of moving that function into some sort of a helper?
The logic and values seem to repeat.

Reply:
Exported to a helper class in `helper` folder inside `qtiCreator`. I choose this folder instead of another helper folder inside `widget > interactions` because is common for both

Updating translations again